samples/bot-sso-adaptivecard/nodejs/README.md

```diff
@@ -0,0 +1,174 @@
+﻿---
+page_type: sample
+description: Microsoft Teams Bot-SSO-Adaptivecard
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "21/01/2023 12:54:21 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-sso-adaptivecard-nodejs
+
+---
+
+# SSO for your Adaptive Cards Universal Actions
+
+This sample code demonstrates how to get enable SSO authentication for your Adaptive Cards Universal Actions in bot.
+
+## Interaction with app
+
+ ![Preview](Images/PreviewAppSSO.gif)
+
```

Comment:
We need to add one more section for demo manifest of C# code:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/636/files#diff-7bb681f0cc7d221ebbf0b17dacc3a1b81bf3bb250a4770a3162eaee026e62e57